src/scheduler/base.py

```diff
@@ -66,6 +67,8 @@ def add_scheduler_job(self, monitor_job, job_id=None, args=None, trigger='interv
         arguments = [monitor_job.run, job_id]
         arguments.extend(args)
 
-        self.add_job(func=self.run_job, id=job_id, args=arguments, trigger=trigger, minutes=minutes)
+        start_date = datetime.now() - timedelta(minutes=(minutes-1)) # start one minute after schedule
```

Comment:
Yea this is a smart move but why choose 1 minute instead of just starting it on creation? We should document this.

Reply:
It's simply a buffer of time between creating and scheduling the job. For all intents and purposes, it can be considered relatively immediate and the user doesn't need to know any different at the moment imo, but the potential to have a start_date that is before the date it is scheduled could mean that it doesn't run depending on the internals of apscheduler, so we avoid that altogether this way